src/Accelerators/NNPA/src/Compiler/DLCompilerUtils.cpp

```diff
@@ -0,0 +1,159 @@
+#include "mlir/Conversion/AffineToStandard/AffineToStandard.h"
```

Comment:
I assume you will do a grep for all ".cpp", ".hpp" files that don't have a copyright. I can do it once this patch is in, if you prefer.

Reply:
will add license in another PR